src/App.tsx

```diff
@@ -1,10 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
+
+  useEffect(() => {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      setPressedKey(event.key);
```

Comment:
you could save this one:
(event: KeyboardEvent) => {
      setPressedKey(event.key);
 into a handler so you dont need to repeat in the return.

Reply:
Totally agree. Thanks.